AllReadyApp/Web-App/AllReady/wwwroot/js/ko.bindings.js

```diff
@@ -0,0 +1,20 @@
+﻿// knockout binding for jquery.maskedinput plugin
+ko.bindingHandlers.masked = {
+    init: function (element, valueAccessor, allBindingsAccessor) {
+        var mask = allBindingsAccessor().mask || {};
+        var placeholder = allBindingsAccessor().placeholder;
+        if (placeholder) {
+            $(element).mask(mask, { placeholder: placeholder, autoclear: false });
+        } else {
+            $(element).mask(mask, { autoclear: false });
+        }
+        ko.utils.registerEventHandler(element, "blur", function () {
+            var observable = valueAccessor();
+            observable($(element).val());
+        });
+    },
+    update: function (element, valueAccessor) {
+        var value = ko.utils.unwrapObservable(valueAccessor());
+        $(element).val(value);
```

Comment:
So I like that you wrapped the `$().mask` plugin with a binding handler. However, the way you've written it you kind of have to implement your own `value` binding handler at the same time. If you search `knockout.debug.js` for `ko.bindingHandlers['value']` you'll see that you probably don't want to do that. I would instead suggest trying to get the mask plugin to play nicely with KO's standard value binding (if you haven't already tried that). There are a couple ways you could do that:
* `masked` could wrap the `value` binding and call through to `ko.bindingHandlers.value.init/update`. In this case I would suggest naming the binding `valueMasked` to more clearly convey that it is derivative of `value`
* `masked` could be its own binding independent of `value` and try not deal with the value of the `input` at all

Additionally, when you have a binding handler that takes multiple options, I think it's better practice for the bound value to be an object containing each of the options, rather than conveying those options through sibling bindings via allBindingsAccessor`: `masked: { placeholder: "", mask: "" }`